sirepo/package_data/template/epicsllrf/lib/epicsConfig-epicsSchema.simple.json

```diff
@@ -1,7 +1,7 @@
 {
     "constants": {
         "epicsModelPrefix": "LLRFSim",
```

Comment:
this file seems to be missing an `epicsModelName` that is resulting in an error in the simple example. The panel in the UI needs this field.